mods/sp/rules/vehicles.yaml

```diff
@@ -1122,16 +1046,19 @@ BIKE:
 		RequiresCondition: DetonateDemoTruck
 	Armament@DemoTruck:
 		Weapon: DemoTruckDummyWeapon
-	AttackOmni:
+	AttackFrontal:
```

Comment:
This one should use attack turreted to make sure it uses oportunity fire, just like the hover truck.

Reply:
we can make the facing tolerance to 128, which can solve this.

Reply:
Fixed.
Use `AttackFrontal` with facing tolerance to 128 has the same effect and save use a trait.

Reply:
Using `FacingTolerance` for these scenarios is the better choice, just keep in mind that when facing requantization hits SP, you need to use 512 - have it hit already?

Reply:
Thanks for mention, and I find it in engine like:
<img width="379" alt="1" src="https://user-images.githubusercontent.com/13763394/99876438-b4024780-2c31-11eb-840b-05f650b9ece9.PNG">
It seems at least the description says it is not hit to SP, for now.

Reply:
The description is wrong then, because `mod.config` states it's using the latest commit on the https://github.com/MustaphaTR/OpenRA/tree/rv-engine branch which is past facing requantization.

Reply:
![1](https://user-images.githubusercontent.com/13763394/99877077-91266200-2c36-11eb-9c9e-67ce83ea075b.gif)
Tested, 128 looks fine.